### ISSUE TITLE
android client ?

### ISSUE THREAD

**Issue by adzweig** [featurecat/lizzie#469]
Hi 
I have tried to use my android tablet as the interface for lizzie running on my PC by exporting the x11 window to my tablet using "metacity". The window opens on the tablet, but it is just white. 
Would there be a way to run Lizzie from an android tablet over local network on a linux PC?

thanks for letting me know.

Adrian

**Comment by featurecat** [comment]
Yes, use Chrome Remote Desktop. I use it all the time, it's a fantastic client.

**Comment by adzweig** [comment]
Thanks a lot for the hint. I'll try it right away!!

Sent from my Sony Xperia™ smartphone

---- Annie Wagner wrote ----

**Comment by bvandenbon** [comment]
or you can use https://www.zbaduk.com which is a webbased client for leela zero.
